src/otx/algo/anomaly/padim.py

```diff
@@ -40,6 +42,7 @@ class Padim(OTXAnomaly, AnomalibPadim):
 
     def __init__(
         self,
+        label_info: LabelInfoTypes = AnomalyLabelInfo(),
```

Comment:
> @chuneuny-emily do you think I need to pass optimizer and scheduler parameters to the model as well? While it is unused in anomaly models, I can see the reference converter
> 
> https://github.com/openvinotoolkit/training_extensions/blob/7f1c7da57365ae362b81542616a8d0ed6517a8f9/src/otx/tools/converter.py#L242

If converter.py is using the "optimizer" and scheduler parameters, you may need to pass these to the model. Otherwise, an error may occur while reading the configuration file.

If there is a part of the code that explicitly sets the learning rate of the optimization function, such as config["model"]["init_args"]["optimizer"]["init_args"]["lr"], this means that the optimization function is set in the configuration file. defined and expected to be used. If there is no "optimizer" or "scheduler" section in the configuration file, the above code may cause an error such as KeyError. @wonjuleee - What do you think?